tests/integration/test_datawarehouse_integration.py

```diff
@@ -94,11 +98,16 @@
             value_error_test_methods += [method]
     if 'filters' in METHOD_PARAMS[method]:
         for param in ('filter_key', 'filter_value'):
-            key_error_test_names += [method + ':' + param]
+            key_error_test_ids += [method + ':' + param]
             (value, match) = KEY_ERROR_TEST_VALUES_AND_MATCHES[param]
             key_error_test_params += [(method, {'filters': value}, match)]
 
 
+with open(Path(expanduser(TOKEN_PATH)), 'r') as token_file:
+    token = token_file.read().replace('\n', '').strip()
+os.environ['XDMOD_API_TOKEN'] = token
```

Comment:
Regarding 106-108, when working through your instructions I had reused an API token from the example notebooks in `xdmod-notebooks`. Those instructions use an env file with the contents `XDMOD_API_TOKEN=<token>" while this loads only the token from a file and directly stores it in the environment variable via python. It practically was not a big deal to figure out the discrepancy, but I just want to advocate for some consistency between `xdmod-notebooks` for token management.

Reply:
Good idea, thanks, I have updated it to use `python-dotenv` like `xdmod-notebooks` does.